app/admin/prizes.rb

```diff
@@ -5,30 +5,28 @@
 
   controller do
     def create
-      total_prizes_percentages = Prize.pluck(:percentage).sum
-      if total_prizes_percentages + params[:prize][:percentage].to_i > 100
+      if Prize.pluck(:percentage).sum + params[:prize][:percentage].to_i > 100
         render json: 'Error! The total percentage is greater than 100%!'
       else
-        Prize.create(prize_params)
-        redirect_to '/admin/prizes'
-      end
+        Prize.create(params.require(:prize).permit(:name, :percentage, :image, :prize_type, :description, :identifier))
+        redirect_to '/admin/prizes' end
```

Comment:
This `end`, and the ones in lines 20, 29 are badly placed. They should be in the line below.

Reply:
But if I change that I'll have this error `Block has too many lines. [27/25] (convention:Metrics/BlockLength)`. 
Can I fix it with this `# rubocop:disable Metrics/BlockLength`?

Reply:
Check if it works on both cases 😅

Reply:
Done